kustomize/aws/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
+  - cache
```

Comment:
When I've tried using folders in a kustomization under `resources`, the command-line kustomize doesn't really like that. It seems to want folders under `bases`, and files under `resources`. Here's a ticket with slightly more detail on the stack-minimal-gcp repo: https://github.com/crossplane/stack-minimal-gcp/issues/20 . I think we probably want to use `bases` for the folders instead of `resources`.

Reply:
Just responded on that issue; `bases` field is [deprecated](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/fields.md#bases) in favor of `resources`.

Reply:
Awesome; less work for us haha - thanks @muvaf !